### ISSUE TITLE
Add cover image of New Jersey to the OCaml Workshop 2020

### ISSUE THREAD

**Issue by ILeandersson** [ocaml/v3.ocaml.org-rescript#442]
Add this picture:
![saketh-garuda-8wqxnCNfwMs-unsplash](https://user-images.githubusercontent.com/73180976/122745837-52669200-d281-11eb-98ea-4b958d2be401.jpg)

to the OCaml Workshop 2020 page so that it looks like this:
<img width="554" alt="Screenshot 2021-06-21 at 11 12 04" src="https://user-images.githubusercontent.com/73180976/122746029-83df5d80-d281-11eb-8ddd-97f179123c21.png">

**Comment by patricoferris** [comment]
Yep sounds good thanks @ILeandersson, this might happen soon or it might wait on https://github.com/ocaml/ood/issues/58 (as we decide how to handle all the media). Separately, isn't this a picture of New York and not New Jersey ? Perhaps we should use something like the "Jersey City from Manhatten" image from the ICFP site https://icfp20.sigplan.org/ :))
